<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sceneRendererDialog</class>
 <widget class="QDialog" name="sceneRendererDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene grid and rendering options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="sceneOptionsGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Scene options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QCheckBox" name="gridOneCheckBox">
        <property name="text">
         <string>Grid #1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="gridTwoCheckBox">
        <property name="text">
         <string>Grid #2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="gridThreeCheckBox">
        <property name="text">
         <string>Grid #3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="colorPickerGridThree">
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/mainToolbar/icons/color_chart.png</normaloff>:/mainToolbar/icons/color_chart.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="colorPickerGridTwo">
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/mainToolbar/icons/color_chart.png</normaloff>:/mainToolbar/icons/color_chart.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="colorPickerGridOne">
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/mainToolbar/icons/color_chart.png</normaloff>:/mainToolbar/icons/color_chart.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="targetDisplayMethodComboBox">
        <item>
         <property name="text">
          <string>Common flow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comet effect</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="targetsDisplayMethodLabel">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>TextLabel: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rendering options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QComboBox" name="backgroundRenderingComboBox">
        <item>
         <property name="text">
          <string>Render everything</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Do not render grid #3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Do not render grid #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Do not render grid #3 and #2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stop bg. rendering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Block scene update</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tappingSeqLabel">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>Tapping sequence:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="renderingEngineLabel">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Rendering engine:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="renderingEngineComboBox">
        <item>
         <property name="text">
          <string>Default Qt painter engine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OpenGL engine</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>sceneRendererDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>sceneRendererDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
